### ISSUE TITLE
Vagrant.has_plugin? command not working

### ISSUE THREAD

**Issue by petems** [WinRb/vagrant-windows#162]
```
$ vagrant plugin list
vagrant-cachier (0.3.3)
vagrant-vbox-snapshot (0.0.4)
vagrant-windows (1.5.0)
```

``` ruby
class String
  def red
    "\033[31m#{self}\033[0m"
  end
  def green
    "\033[32m#{self}\033[0m"
  end
end

#Vagrantfile
if Vagrant.has_plugin?('vagrant-windows')
  puts "Vagrant windows found, no action needed".green
else
  puts "vagrant-windows missing! Please install Vagrant windows!".red
  puts "Run this command in your terminal:".red
  puts "vagrant plugin install vagrant-windows".green
  exit 1
end

# Rest of standard Vagrantfile
```

```
$ vagrant status
vagrant-windows missing! Please install Vagrant windows!
Run this command in your terminal:
vagrant plugin install vagrant-windows
```

**Comment by sneal** [comment]
Indeed, the registered plugin name is not what users would expect. I'll have a fix shortly.

**Comment by sneal** [comment]
Fixed in 1.5.1
